nautobot/core/api/utils.py

```diff
@@ -386,6 +386,10 @@ def get_m2m_and_non_m2m_fields(self):
 
         return m2m_fields, non_m2m_fields
 
+    def capitalize(self, value: str):
+        """Capitalize value only if not uppercase."""
+        return value if (value.isupper()) else value.capitalize()
```

Comment:
This won't work as well for cases like `IP Address`

Reply:
Looks like my comment on the commit itself isn't showing up here - should be able to use the existing `bettertitle` function for this purpose.